packages/go/ein/azure.go

```diff
@@ -1482,3 +1483,14 @@ func getKeyVaultPermissions(data models.KeyVaultAccessPolicy) []graph.Kind {
 	}
 	return relationships
 }
+
+func ResourceWithinScope(resource, scope string) bool {
+	if strings.EqualFold(resource, scope) {
+		return true
+	}
+	resourceGroupLevel, _ := regexp.MatchString("^[\\w\\d\\-\\/]*/resourceGroups/[0-9a-zA-Z]+$", resource)
```

Comment:
Compile this regex and assign it to an outer `var` - it will avoid dumping the error since `MatchString` on the compiled regex object will only return `bool`.

Reply:
Thanks - pushed the requested change